.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
@howard 
Do we actually need to specifically set the distro value to `trusty` as it became the default value today?

Reply:
Explicitly setting the version protects us from discrepancies between builds of different contributors in the future when another update of the default infrastructure comes along. Of course, this value needs changing in the long term when the next system-wide update comes along, but in any case, explicitly setting versions helps consistency and the ability to reproduce issues.